app/Http/ViewComposers/ProfileComposer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\ViewComposers;
+
+use Illuminate\View\View;
+// use App\Repositories\Contracts\OptionRepository;
+use App\Eloquent\Option;
+
+class ProfileComposer
+{
+
+    /**
+     * Create a new profile composer.
+     *
+     * @return  void
+     */
+    // protected $option;
+
+    // public function __construct(
+    // )
+        // OptionRepository $option
+    // ) {
+    //     $this->option = $option;
+    // }
+
+    /**
+     * Bind data to the view.
+     *
+     * @param    View  $view
+     * @return  void
+     */
+    public function compose(View $view)
+    {
+        // truyền dữ liệu ở đây nè
+        // $view->with('option', $this->option->setting());
+        // dd($this->option->setting());
+        // $data['textFooters'] = Option::where(['key' => 'text_footer'])->get();
+        // $data['banners'] = Option::where(['key' => 'banner'])->get();
+        // $data['bannerBooks'] = Option::where(['key' => 'banner_book'])->first();
+        // $data['apps'] = Option::where(['key' => 'app'])->get();
+        // $data['textBanners'] = Option::where(['key' => 'text_banner'])->get();
+        // $data['textApps'] = Option::where(['key' => 'app_text'])->get();
+        // $data['contacts'] = Option::where(['key' => 'contact'])->get();
+        // $data['address'] = Option::where(['key' => 'address'])->get();
+        // $data['emails'] = Option::where(['key' => 'email'])->get();
+        // // dd($data);
+        
+        // view()->share($data);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Http/ViewComposers/ProfileComposer.php</br>**Line**: 50</br>Expected 1 newline at end of file; 0 found